<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>788</width>
    <height>408</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Nhân Sự</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QMdiArea" name="mdiArea"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>788</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCai_dat">
    <property name="title">
     <string>&amp;Cài đặt</string>
    </property>
    <addaction name="actionSQL"/>
   </widget>
   <widget class="QMenu" name="menuHe_thong">
    <property name="title">
     <string>Hệ thố&amp;ng</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionQuyen"/>
    <addaction name="actionTai_Khoan"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuTh_ng_tin">
    <property name="title">
     <string>Th&amp;ông tin</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="separator"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <widget class="QMenu" name="menuB_o_c_o">
    <property name="title">
     <string>Báo cáo</string>
    </property>
    <addaction name="actionH_a_n"/>
    <addaction name="actionL_ng_theo_ph_ng"/>
   </widget>
   <widget class="QMenu" name="menuQuan_ly">
    <property name="title">
     <string>Quản lý</string>
    </property>
   </widget>
   <addaction name="menuCai_dat"/>
   <addaction name="menuHe_thong"/>
   <addaction name="menuQuan_ly"/>
   <addaction name="menuB_o_c_o"/>
   <addaction name="menuTh_ng_tin"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionDan_nhap"/>
   <addaction name="separator"/>
   <addaction name="actionNhanVien"/>
   <addaction name="actionNhapHang"/>
   <addaction name="separator"/>
   <addaction name="actionQuan_Ly_Luong"/>
  </widget>
  <action name="actionSQL">
   <property name="text">
    <string>&amp;SQL</string>
   </property>
  </action>
  <action name="actionDan_nhap">
   <property name="text">
    <string>Đăng nhập</string>
   </property>
  </action>
  <action name="actionNhanVien">
   <property name="text">
    <string>DS Nhân viên</string>
   </property>
   <property name="toolTip">
    <string>Danh sách sinh viên cho từng phòng</string>
   </property>
  </action>
  <action name="actionNhapThuoc">
   <property name="text">
    <string>Nhập thuốc</string>
   </property>
   <property name="toolTip">
    <string>From nhap hang</string>
   </property>
  </action>
  <action name="actionTC_cho_tr_em">
   <property name="text">
    <string>&amp;TC cho trẻ em</string>
   </property>
  </action>
  <action name="actionBenh_truyen_nhiem">
   <property name="text">
    <string>&amp;Bệnh truyền nhiễm</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>&amp;About Qt</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionQuan_ly_NKH">
   <property name="text">
    <string>Nhóm khách hàng</string>
   </property>
  </action>
  <action name="actionNhapHang_2">
   <property name="text">
    <string>Nhập Hàng</string>
   </property>
  </action>
  <action name="actionQuyen">
   <property name="text">
    <string>Quyền </string>
   </property>
   <property name="toolTip">
    <string>Phân quyền cho hệ thống</string>
   </property>
  </action>
  <action name="actionTai_Khoan">
   <property name="text">
    <string>Tài Khoản </string>
   </property>
  </action>
  <action name="actionPhieu_chi">
   <property name="text">
    <string>Phiếu chi</string>
   </property>
  </action>
  <action name="actionKhach_Hang">
   <property name="text">
    <string>Khách Hàng</string>
   </property>
  </action>
  <action name="actionGia">
   <property name="text">
    <string>Giá</string>
   </property>
   <property name="toolTip">
    <string>Quản lý giá thuốc</string>
   </property>
  </action>
  <action name="actionNhapHang">
   <property name="text">
    <string>Chấm công</string>
   </property>
  </action>
  <action name="actionH_a_n">
   <property name="text">
    <string>Lương nhân viên</string>
   </property>
  </action>
  <action name="actionNhom_Hang_Hoa">
   <property name="text">
    <string>Nhóm Hàng Hóa</string>
   </property>
  </action>
  <action name="actionHang_Hoa">
   <property name="text">
    <string>Hàng Hóa</string>
   </property>
  </action>
  <action name="actionBH">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Bảo hiểm</string>
   </property>
  </action>
  <action name="actionTuyen">
   <property name="text">
    <string>Tuyển NV</string>
   </property>
  </action>
  <action name="actionQuan_Ly_Luong">
   <property name="text">
    <string>Tính lương</string>
   </property>
   <property name="toolTip">
    <string>Chức năng tính lương dựa theo kỳ, in báo cáo lương </string>
   </property>
  </action>
  <action name="actionHop_dong">
   <property name="text">
    <string>Hợp đồng</string>
   </property>
  </action>
  <action name="actionHoc_thu_viec">
   <property name="text">
    <string>Học thử việc</string>
   </property>
  </action>
  <action name="actionChuc_vu">
   <property name="text">
    <string>Chức vụ</string>
   </property>
  </action>
  <action name="actionL_ng_theo_ph_ng">
   <property name="text">
    <string>Lương theo phòng</string>
   </property>
  </action>
  <action name="actionTuy_n_nh_n_vi_n">
   <property name="text">
    <string>Tuyển nhân viên</string>
   </property>
  </action>
  <action name="actionTh_ng_n_h_ng">
   <property name="text">
    <string>Quyết định</string>
   </property>
  </action>
  <action name="actionB_o_tr">
   <property name="text">
    <string>Bảo trì</string>
   </property>
  </action>
  <action name="actionNgh_l_u">
   <property name="text">
    <string>Nghỉ lưu</string>
   </property>
  </action>
  <action name="actionImport_hop_dong">
   <property name="text">
    <string>Import hop dong</string>
   </property>
  </action>
  <action name="actionMo_form_bao_tri">
   <property name="text">
    <string>Bảo tri dữ liệu</string>
   </property>
  </action>
  <action name="actionT_m_ng">
   <property name="text">
    <string>Tạm ứng</string>
   </property>
  </action>
  <action name="actionThuong">
   <property name="text">
    <string>Thưởng</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
